flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/BlocklistDeclarativeSlotPoolTest.java

```diff
@@ -299,14 +308,18 @@ public BlocklistDeclarativeSlotPoolBuilder setBlockedTaskManagerChecker(
             return this;
         }
 
-        public BlocklistDeclarativeSlotPool build() {
+        public BlocklistDeclarativeSlotPool build(
+                Duration slotRequestMaxInterval,
+                ComponentMainThreadExecutor componentMainThreadExecutor) {
```

Comment:
In general, the build method without any parameter, and all parameters are set by setter.